package.json

```diff
@@ -91,6 +92,7 @@
     "babel": "5.8.38",
     "babel-core": "5.8.38",
     "babel-loader": "5.3.2",
+    "babel-polyfill": "6.9.1",
```

Comment:
Great catch man, looks like this isn't needed at all since we're using 5, not 6.